src/test/java/com/booking/superride/controller/RideControllerTest.java

```diff
@@ -6,26 +6,23 @@
 import static org.hamcrest.Matchers.*;
 import static org.junit.jupiter.api.Assertions.*;
 
-import com.booking.superride.common.ContainersConfig;
+import com.booking.superride.common.AbstractIntegrationTest;
 import com.booking.superride.domain.RideBookingRequest;
 import com.booking.superride.domain.RideDetailsResponse;
 import com.booking.superride.entity.CustomerDetails;
 import com.booking.superride.entity.TaxiDetails;
 import com.booking.superride.repository.CustomerRepository;
 import com.booking.superride.repository.RideRepository;
 import com.booking.superride.repository.TaxiRepository;
+import io.restassured.RestAssured;
 import io.restassured.http.ContentType;
 import java.time.LocalDateTime;
 import org.junit.jupiter.api.*;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.context.annotation.Import;
 
-@Import(ContainersConfig.class)
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
```

Comment:
@agniraj09 , Please check if we can move this line to `AbstractIntegrationTest.java`